### PR TITLE
VSCode Release v0.3.0

### PR DESCRIPTION
**Pull request by shonfeder**

### CODE REVIEW

**Comment by shonfeder** [comment]
The empty changelog is unfortunate. #701 should help address, but in the meantime, is Ok if we must accept the slight oddity? I'd rather not have to do busy work just cause or release process for the vscode plugin is a bit wonky :)  Tho i will if we think it's important. WDYT, @bugarela ?

**Comment by bugarela** [comment]
It would be nice to automatically add a `update quint from x to y` entry to the changelog, shouldn't be too hard to do. I'd suggest making a manual edit to the changelog adding a note like that now, it shouldn't break the release since it is not very git-dependent afaict.

**Review by bugarela — APPROVED** [review]

File: vscode/quint-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 ### Security
 
+## v0.3.0 -- 2023-04-03
+
+### Added
+### Changed
+
+- Update quint from `0.2.1` to `0.3.0`
```
Comment (shonfeder) [inline vscode/quint-vscode/CHANGELOG.md:22]:
```suggestion
- Update quint from `0.2.1` to `0.3.0`. See #768.
```

**Comment by shonfeder** [comment]
Sounds good. Made that change. Thanks for the :heavy_check_mark: !

**Pull request merged**